src/cpu/base.hh

```diff
@@ -621,7 +632,13 @@ class BaseCPU : public ClockedObject
     }
 
     static Counter
-    numSimulatedOps()
+    numSimulatedInsts()
+    {
+      return totalNumSimulatedInsts() - (globalStats -> previousInsts);
+    }
```

Comment:
Shouldn't we make this a method of GlobalStats? In this way we avoid extra-stat usage of it